.gitignore

```diff
@@ -122,3 +122,92 @@ flake8_junit.xml
 
 .ipython
 .matplotlib
+figures/individual_viral_dynamics/viral_dynamics_agent_0.png
```

Comment:
I'd suggest just adding `*.png` rather than naming individual files

Reply:
Done - also since the scope of `.gitignore` files is everything under the directory, we can have multiple `.gitignore` files e.g., if we want to name specific files inside the `analyses` folder that are too specific for the main STIsim file - I've added an example of that too (cc @alina-muellenmeister)

Reply:
Thank you, Romesh!